kvwrapper_etcd_v3/kvwrapper_etcd_v3.go

```diff
@@ -0,0 +1,180 @@
+package kvwrapper_etcd_v3
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	// "github.com/behance/go-common/kvwrapper"
+	"github.com/behance/go-common/kvwrapper"
+	log "github.com/behance/go-logging/log"
+	etcdv3 "github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/etcdserver/api/v3rpc/rpctypes"
+	// "google.golang.org/grpc"
```

Comment:
Minor - let's remove the commented out packages

Reply:
When I ran go test -v ./... , I indeed started and etcd locally.  This doesn't take place in CI

Reply:
The CI failure would've occurred in the existing V2 implementation, too.

Reply:
Correct - for a unit test, it is not a reasonable expectation that the user should be running a localhost version of an external service - hence the tests should be included, should the user want to validate the behavior, but should not be enabled by default. We want to always keep the CI passing - I think this is an improvement over V2 since there are repeatable tests.

Reply:
My previous statement is misleading.  V2 had no tests in place requiring a live etcd server, using  KVFaker, instead.  I introduced tests for v3, that actually test against a local etcd server.